rts/Rendering/Textures/TextureFormat.cpp

```diff
@@ -124,3 +150,44 @@ GLenum GL::GetBindingQueryFromTarget(GLenum target) {
 	}
 	return 0;
 }
+
+GLsizei GL::GetPixelFormatSize(GLenum format) {
+	GLsizei size;
+	switch (format) {
+		case GL_RED:
+		case GL_RED_INTEGER:
+		case GL_GREEN:
+		case GL_GREEN_INTEGER:
+		case GL_BLUE:
+		case GL_BLUE_INTEGER:
+		case GL_ALPHA:
+		case GL_ALPHA_INTEGER:
+		case GL_LUMINANCE:
+		case GL_COLOR_INDEX:
+		case GL_STENCIL_INDEX:
+		case GL_DEPTH_COMPONENT: {
+			size = 1;
+		} break;
+		case GL_RG:
+		case GL_RG_INTEGER:
+		case GL_LUMINANCE_ALPHA: {
+			size = 2;
+		} break;
+		case GL_RGB:
+		case GL_RGB_INTEGER:
+		case GL_BGR:
+		case GL_BGR_INTEGER: {
+			size = 3;
+		} break;
+		case GL_RGBA:
+		case GL_RGBA_INTEGER:
+		case GL_BGRA:
+		case GL_BGRA_INTEGER: {
+			size = 4;
+		} break;
+		default: {
+			size = 0;
```

Comment:
Add assert(false) for the default case.

Reply:
This is a utility function. Assert must or must not be placed in a function that uses it.